src/internal/Popover.js

```diff
@@ -310,6 +310,8 @@ export default class Popover extends Component {
     };
   }
 
+  handleGetOffsetTop = getOffsetTop;
```

Comment:
The reason I went this way was that by exporting the functions we are declaring them to be "official" members of the component, in fact they are internal logical routers. From the perspective of writing tests absolutely exporting. From a program interface I think this is the better way to go. No?